application/testing/CMakeLists.txt

```
@@ -803,6 +803,7 @@ f3d_test(NAME TestInteractionZoomToggleOrthographicProjection DATA cow.vtp INTER
 f3d_test(NAME TestInteractionRotateCameraMinus90 DATA f3d.glb INTERACTION)
 f3d_test(NAME TestInteractionRotateCamera90 DATA f3d.glb INTERACTION)
 f3d_test(NAME TestInteractionPanWithShift DATA f3d.glb INTERACTION) #Shift;LeftMouse;MouseMovements
+f3d_test(NAME TestInteractionFileOpen DATA cow.vtp INTERACTION NO_BASELINE)
```

See previous suggestion
```suggestion
f3d_test(NAME TestInteractionFileOpen DATA cow.vtp INTERACTION NO_BASELINE ENV CTEST_OPEN_DIALOG_FILE="${F3D_SOURCE_DIR}/testing/data/cow.vtp")
```

---

implemented